spec/system/case_contacts/index_spec.rb

```diff
@@ -1,28 +1,41 @@
 require "rails_helper"
 
 RSpec.describe "case_contacts/index", :disable_bullet, type: :system do
-  let(:organization) { create(:casa_org) }
   let(:volunteer) { create(:volunteer, display_name: "Bob Loblaw", casa_org: organization) }
-  let(:casa_case) { create(:casa_case, casa_org: organization, case_number: "CINA-1") }
-  let!(:case_assignment) { create(:case_assignment, volunteer: volunteer, casa_case: casa_case) }
-  let!(:case_contact) { create(:case_contact, creator: volunteer, casa_case: casa_case) }
+  let(:organization) { create(:casa_org) }
 
-  before(:each) do
-    sign_in volunteer
-    visit case_contacts_path
-  end
+  context "with case contacts" do
+    let(:casa_case) { create(:casa_case, casa_org: organization, case_number: "CINA-1") }
+    let!(:case_assignment) { create(:case_assignment, volunteer: volunteer, casa_case: casa_case) }
+    let!(:case_contact) { create(:case_contact, creator: volunteer, casa_case: casa_case) }
```

Comment:
It would be cleaner to handle this with a trait `create(:casa_case, :with_case_contacts)`

Sadly `trait :with_case_contacts` has not been written
it would make for a good beginner issue

`casa/spec/factories/casa_case.rb`

Reply:
I'm gonna go ahead and make an issue of this

Reply:
Oh I cannot as the text is not merged yet

Reply:
It's probably better just to change this to support localization